detect.go

```diff
@@ -23,18 +24,21 @@ type BuildPlanMetadata struct {
 // Detect will return a packit.DetectFunc that will be invoked during the
 // detect phase of the buildpack lifecycle.
 //
-// This buildpack has a less-common provides/requires structure. There are two
-// main requirement groups:
-// One for HTTPD in which "php", "php-fpm", "httpd"
-// and "httpd-config" are required at launch-time.
-// The second one for for Nginx in which "php", "php-fpm", "nginx"
-// and "nginx-config" are required at launch-time.
-
-// This buildpack will always detect, and in the case of HTTPD, the buildpack
-// will provide and require `httpd-start`. In the case of Nginx, the buildpack
-// will provide and require `nginx-start`. This is unusual, but will allow the
-// buildpack Build function access to which web server start command is needed,
-// since the requirements are not easily checked otherwise.
```

Comment:
Why is all of this context removed?

Reply:
The buildpack no longer has any reference to `httpd-start` or `nginx-start`, so the entire block was simply incorrect.